src/components/FxTokens.ts

```diff
@@ -19,12 +18,12 @@ export type FxTokensConfig = {
   graphEndpoint: string;
 };
 
-type FxTokenMulticallMulticall = {
+type Price = {
```

Comment:
```suggestion
type FxTokenMulticall = {
```
It is used in a multicall and might return more than just the price in the future

Reply:
this type is actually not used anywhere, so I have removed it. The type FxToken now includes a price.

Reply:
changed to FxTokenMulticallResponse